frontend/src/metabase/ui/utils/colors.ts

```diff
@@ -59,3 +59,8 @@ export function getThemeColors(): Record<string, ColorShades> {
     ),
   };
 }
+
+export type MantineColor =
```

Comment:
I couldn't find a type that worked (there is the `MantineColors` type that's included with Mantine but it's just a straight-up `string` which isn't helpful for auto-complete)